README.md

```diff
@@ -22,9 +22,20 @@ export interface Options {
   isProduction?: boolean
 
   // options to pass on to vue/compiler-sfc
-  script?: Partial<SFCScriptCompileOptions>
-  template?: Partial<SFCTemplateCompileOptions>
-  style?: Partial<SFCStyleCompileOptions>
+  script?: Partial<Pick<SFCScriptCompileOptions, 'babelParserPlugins'>>
+  template?: Partial<
+    Pick<
+      SFCTemplateCompileOptions,
+      | 'compiler'
+      | 'compilerOptions'
+      | 'preprocessOptions'
+      | 'transpileOptions'
+      | 'transformAssetUrls'
+      | 'transformAssetUrlsOptions'
+      | 'isTS'
```

Comment:
Do we want to expose `isTS`?

Reply:
I think we can skip `isTS` as it's not used anywhere in the Vue 2 codebase.
I guess this feature was dropped during the Vue 2.7 development.

(`isTS` is used in Vue 3 for template expression TypeScript syntax support, but 2.7 release notes explicitly said this feature is not supported. Anyway, this option makes no sense to end users)

Reply:
I see. I've removed it 👍